website/source/docs/guides/windows-guide.html.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: "docs"
+page_title: "Windows Service"
+sidebar_current: "docs-guides-windows-service"
+description: |-
+  For our friends running Consul on Windows, we have good news. By using the _sc_ command either on Powershell or 
+  the Windows command line, you can make Consul run as a service. For more details about the _sc_ command
+  the Windows page for [sc](https://msdn.microsoft.com/en-us/library/windows/desktop/ms682107(v=vs.85).aspx)
+  should help you get started.
+
+---
+
+# Overview
+For our friends running Consul on Windows, we have good news. By using the _sc_ command either on Powershell or 
+the Windows command line, you can make Consul run as a service. For more details about the _sc_ command
+the Windows page for [sc](https://msdn.microsoft.com/en-us/library/windows/desktop/ms682107(v=vs.85).aspx)
+should help you get started.
+
+Please remember to create a permanent directory for storing the configuration files,
+as it would be handy, if you're starting Consul with the _-config-dir_ argument. 
+
+The steps presented here assume, that the user has launched **Powershell** with _Adminstrator_ capabilities.
+
+If you come across bugs while using Consul for Windows, do not hesitate to open an issue [here](https://github.com/hashicorp/consul/issues).
+
+## Detailed steps involved in making Consul run as a service on Windows
+Download the Consul binary for your architecture.
+
+Setup your environmental _path_ variable, so that Windows can find 
+the Consul binary. (Will be handy for quick Consul commands)
```

Comment:
"handy" -> "useful" 

To me useful sounds like a word more likely to be common for our international audience. I could be wrong though.